ui/packages/workbench/package.json

```diff
@@ -42,7 +42,8 @@
 		"@gradio/tabs": "workspace:^0.0.1",
 		"@gradio/theme": "workspace:^0.0.1",
 		"@gradio/upload": "workspace:^0.0.1",
+		"@gradio/uploadbutton": "workspace:^0.0.1",
 		"@gradio/video": "workspace:^0.0.1",
-		"@gradio/uploadbutton": "workspace:^0.0.1"
```

Comment:
I added this cause I got `missing peer dependencies` errors in pnpm. Not sure if it's actually needed